corehq/motech/repeaters/migrations/0002_repeaters_db.py

```diff
@@ -0,0 +1,170 @@
+# Generated by Django 3.2.16 on 2023-04-12 20:05
+#
+# This migration assumes that repeaters_repeatrecord and
+# repeaters_repeatrecordattempt tables are not in use.
+
+from django.conf import settings
+from django.db import connections, migrations
+from django.db.utils import DEFAULT_DB_ALIAS
+
+# Use a separate user for the fdw connection because postgres_fdw
+# changes session settings like search_path that interfere with other
+# clients because of pg_bouncer connection pooling.
+# https://www.postgresql.org/docs/9.4/postgres-fdw.html#AEN130855
+# PgBouncer maintains a pool of connections for each unique user,
+# database pair.
+# https://pgdash.io/blog/pgbouncer-connection-pool.html
+
+MOVE_TO_SEPARATE_DB = """
+    CREATE SERVER repeaters_fdw FOREIGN DATA WRAPPER postgres_fdw
+        OPTIONS (host '{HOST}', port '{PORT}', dbname '{NAME}');
+    CREATE USER MAPPING FOR {USER} SERVER repeaters_fdw
+        OPTIONS (user 'repeaters_fdw_user', password '{PASSWORD}');
+    GRANT USAGE ON FOREIGN SERVER repeaters_fdw TO {USER};
+
+    DROP TABLE repeaters_repeatrecordattempt;
+    DROP TABLE repeaters_repeatrecord;
```

Comment:
Is this safe because there is no data in these yet?

Reply:
Correct.